src/main/java/homework_2/RandomCharsTable.java

```diff
@@ -0,0 +1,91 @@
+package homework_2;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.Random;
+import java.util.StringJoiner;
+
+public class RandomCharsTable {
+
+    public static void main(String[] args) {
+        try {
+            String inputString = readInputFromConsole();
+
+            if (inputString.matches("[1-9]\\s+[1-9]\\s+even|\\s*[1-9]\\s*[1-9]\\s*odd")) {
```

Comment:
Opt: you can simplify groping in regex by adding ( and )

Reply:
done